opentelemetry-api/src/opentelemetry/util/types.py

```diff
@@ -13,8 +13,11 @@
 # limitations under the License.
 
 
-from typing import Mapping, Optional, Sequence, Tuple, Union
+from typing import Dict, List, Mapping, Optional, Sequence, Tuple, Union
 
+AnyValue = Union[
```

Comment:
Nit: Might be nice to add a comment to link to [this](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#type-any) in the specs.

Reply:
Done